src/commands/moderation/LockCommand.js

```diff
@@ -57,8 +58,8 @@ class LockCommand extends Command {
         if (channels.length === 0) return;
 
         {
-            const regex = new RegExp(`${this.prefix}${this.name} (<?#?\\d+>?)+`);
-            const start = this.message.content.match(regex)[0].length + 1;
+            const regex = new RegExp(`${this.prefix}${this.name} (<?#?\\d+>? ?)+`);
```

Comment:
It might be helpful to escape the strings inserted into the regex filter or to not insert strings into the regex filter at all.
Especially the prefix could be a character with a special meaning in regex.

You could, for example, only try to match `this.message.content.substr(this.prefix.length + this.name.length + 1)`